shop/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+
+# H2
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console
+# Datasource
+spring.datasource.url=jdbc:h2:~/Documents/App/online-shop-RaulDan/shop
+spring.datasource.username=sa
+spring.datasource.password=
+spring.flyway.user=sa
+spring.flyway.password=
+spring.flyway.url=jdbc:h2:~/Documents/App/online-shop-RaulDan/shop
```

Comment:
lets use a project related path. The project has to be able to be cloned and started up by anyone, without further tweaks, and not everyone has the folder App in their Documents file 📁